### PR TITLE
Name remaining world event

### PR DESCRIPTION
**Pull request by IotaBread**

### CODE REVIEW

File: mappings/net/minecraft/world/WorldEvents.mapping [hunk 1]
```diff
@@ -343,3 +343,9 @@ CLASS net/minecraft/class_6088 net/minecraft/world/WorldEvents
 		COMMENT A Bat takes off.
 		COMMENT <br>Plays the bat take off sound event.
 		COMMENT <p>Called by {@link net.minecraft.entity.passive.BatEntity#mobTick() BatEntity#mobTick}
+	FIELD field_33511 BONE_MEAL_USED I
+		COMMENT Bone meal is used
+		COMMENT <br>Plays the bone meal item used sound event and spawns happy villager particles
+		COMMENT <p> Called by {@link net.minecraft.item.BoneMealItem#useOnBlock(net.minecraft.item.ItemUsageContext) BoneMealItem#useOnBlock},
```
Comment (liach) [inline mappings/net/minecraft/world/WorldEvents.mapping]:
```suggestion
		COMMENT <p>Called by {@link net.minecraft.item.BoneMealItem#useOnBlock(net.minecraft.item.ItemUsageContext) BoneMealItem#useOnBlock},
```

**Review by liach — COMMENTED** [review]
Please add this to the unpick config, thanks!

**Comment by liach** [comment]
For the unpick config, add a line after this one https://github.com/FabricMC/yarn/blob/c13a13d765d0798376f77d8aff41d72cf70a1199/unpick-definitions/world_events.unpick#L72 and add in a similar fashion (you just need to change field name)

**Comment by IotaBread** [comment]
Accidentally pushed with extra changes...

File: mappings/net/minecraft/world/WorldEvents.mapping [hunk 2]
```diff
@@ -343,3 +343,9 @@ CLASS net/minecraft/class_6088 net/minecraft/world/WorldEvents
 		COMMENT A Bat takes off.
 		COMMENT <br>Plays the bat take off sound event.
 		COMMENT <p>Called by {@link net.minecraft.entity.passive.BatEntity#mobTick() BatEntity#mobTick}
+	FIELD field_33511 BONE_MEAL_USED I
+		COMMENT Bone meal is used
+		COMMENT <br>Plays the bone meal item used sound event and spawns happy villager particles
```
Comment (liach) [inline mappings/net/minecraft/world/WorldEvents.mapping]:
```suggestion
		COMMENT Bone meal is used.
		COMMENT <br>Plays the bone meal item used sound event and spawns happy villager particles.
```

**Review by liach — APPROVED** [review]

**Pull request merged**